File: Test/Platform/Tests/CLR/mscorlib/exceptions/VB/Master.vb
Option Explicit On
Option Strict On

Imports System.Diagnostics
Imports Microsoft.SPOT.Platform.Test

Namespace Microsoft.SPOT.Platform.Tests

    Public Class Master
        Shared Sub Main()

            Dim args() As String = { "ExceptionTest", "AppDomainExceptionTest" }

            Dim runner As MFTestRunner = New MFTestRunner(args)

        End Sub

    End Class

End Namespace

File: Test/Platform/Tests/Desktop/MFDeploy/MFDeployBasicTests/Master.cs
using System;
using Microsoft.SPOT.Platform.Test;

namespace Microsoft.SPOT.Platform.Tests
{
    class Master
    {
        static void Main()
        {
            string[] args = { "FeatureTests" };
            MFDesktopTestRunner runner = new MFDesktopTestRunner(args);
        }
    }
}
